typed/clj.analyzer/src/typed/clj/analyzer/utils.cljc

```diff
@@ -435,14 +435,14 @@
   (eduction (remove (comp :static :flags)) (members2 class f)))
 
 (defn static-methods [class method argc]
-  (into [] (filter #(and (instance? clojure.reflect.Method %)
```

Comment:
Is a keyword lookup faster than `instance?`?

Reply:
It's not, just looks a bit cleaner. But might as well leave it as is.